modular_skyrat/modules/loadouts/loadout_items/loadout_datum_neck.dm

```diff
@@ -131,10 +127,6 @@ GLOBAL_LIST_INIT(loadout_necks, generate_loadout_items(/datum/loadout_item/neck)
 	name = "Choker"
 	item_path = /obj/item/clothing/neck/human_petcollar/choker
 
-/datum/loadout_item/neck/thinchoker
```

Comment:
This too.

Reply:
i have no idea where these came from tbh lemme revert it i think this might have been an issue from when i synced my repo